From 4574d26bb73b72838902edf29b02b8e252a31686 Mon Sep 17 00:00:00 2001
From: Wang Quanyang <quanyang.wang@windriver.com>
Date: Thu, 8 Oct 2015 10:12:52 +0000
Subject: [PATCH 1008/1008] mtd/spi: support en25s64 spi-nor device

In upstream commit a5b7616c55e1
("mtd: m25p80,spi-nor: Fix module aliases for m25p80"),
the id_table in spi_driver struct has changed from "spi_nor_ids"
to "m25p_ids". So need add "en25s64" to the new id_table.

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/mtd/devices/m25p80.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mtd/devices/m25p80.c b/drivers/mtd/devices/m25p80.c
index 3af137f..5538e5d 100644
--- a/drivers/mtd/devices/m25p80.c
+++ b/drivers/mtd/devices/m25p80.c
@@ -265,7 +265,7 @@ static const struct spi_device_id m25p_ids[] = {
 	{"at25df641"},	{"at26f004"},	{"at26df081a"},	{"at26df161a"},
 	{"at26df321"},	{"at45db081d"},
 	{"en25f32"},	{"en25p32"},	{"en25q32b"},	{"en25p64"},
-	{"en25q64"},	{"en25qh128"},	{"en25qh256"},
+	{"en25q64"},	{"en25qh128"},	{"en25qh256"},	{"en25s64"},
 	{"f25l32pa"},
 	{"mr25h256"},	{"mr25h10"},
 	{"gd25q32"},	{"gd25q64"},
-- 
2.0.2

